R/imageutils.R

```diff
@@ -4,14 +4,43 @@ startPNG <- function(filename, width, height, res, ...) {
   # Finally, if neither quartz nor Cairo, use png().
   if (capabilities("aqua")) {
     pngfun <- grDevices::png
-  } else if ((getOption('shiny.usecairo') %OR% TRUE) &&
-      nchar(system.file(package = "Cairo"))) {
+  } else if ((getOption('shiny.useragg') %OR% FALSE) && is_available("ragg")) {
```

Comment:
A few things:

* This can be first in the if-else, so that Mac users can use ragg if they really want.
* We don't document this option since it's just for experimentation for now, and will be superseded by a different one in the future. It also should have a comment stating as much.
* We can change to `getOption('shiny.useragg', default = FALSE)` here.